ondewo/nlu/intent.proto

```diff
@@ -968,4 +971,30 @@ message DeleteTrainingPhraseRequest {
     // Required. The name of the training phrase.
     // Format: `projects/<Project ID>/agent/intents/<Intent ID>/trainingPhrases/<Training Phrase ID>`
     string name = 1;
+}
+
+// The request message for TrainingPhraseRequest
+message ListTrainingPhrasesRequest {
+    // Required. The agent to list all intents from.
+    // Format: `projects/<Project ID>/agent/intents/<Intent ID>`.
+    string intent_name = 1;
+
+    // Optional. The language to list training phrases, parameters and rich
+    // messages for. If not specified, the agent's default language is used.
+    string language_code = 2;
+
+    // Optional. The next_page_token value returned from a previous list request.
+    string page_token = 3;
```

Comment:
I know this is not in other protos, but it would be great to have an example of a page_token in the comments for documentation and easier usage.

Reply:
Agree, I'll add that!